update.py

```diff
@@ -15,13 +15,21 @@
 
 VARIANTS = [
     {
-        "name": "ubuntu",
+        "name": "ubuntu1604",
         "parent": "ubuntu",
     },
     {
-        "name": "alpine",
+        "name": "ubuntu1804",
+        "parent": "ubuntu"
+    },
+    {
+        "name": "alpine311",
```

Comment:
even more variants ;-p

LTS ubuntu I guess, hard to argue, but two versions of alpine 3 ?

Reply:
I'm just about to push up a change to _not_ build the oldver variants _most_ of the time, ie, replacing te logic which currently rewrites the file in python by just replacing strings.

Seems more stable to have a declaration of, say, how a alpine 3.8 image should be built, and if alpine 3.11 doesn't work then just use that, as opposed to trying to basically sed and grep through the file to update it, so I am trimming down the number of variants :)